app/WebParsing/ReqParser.hs

```diff
@@ -7,8 +8,17 @@ import Database.Requirement
 
 -- define separators
 fromSeparator :: Parser ()
-fromSeparator = Parsec.spaces >> (Parsec.try (Parsec.string "FCE")
-             <|> (Parsec.string "FCEs")) >> Parsec.spaces
+fromSeparator = Parsec.spaces >> (Parsec.choice $ map Parsec.try (map Parsec.string [
```

Comment:
Note that you can merge two calls to `map`: `map f (map g lst) == map (f . g) lst`